apps/opik-backend/src/main/resources/liquibase/db-app-state/migrations/000006_add_projects_last_updated_trace_at.sql

```diff
@@ -0,0 +1,6 @@
+--liquibase formatted sql
+--changeset idoberko2:add_projects_last_updated_trace_at
+
+ALTER TABLE projects ADD COLUMN last_updated_trace_at TIMESTAMP(6);
```

Comment:
> TIMESTAMP has a default of 0 unless defined with the NULL attribute, in which case the default is NULL .

https://dev.mysql.com/doc/refman/8.4/en/timestamp-initialization.html#:~:text=TIMESTAMP%20has%20a%20default%20of,case%20the%20default%20is%20NULL%20.

Better define NULL as the default value.

Reply:
Done